misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/utils/PackageUtils.java

```diff
@@ -86,4 +89,24 @@ public static boolean isBlank(String str) {
         }
         return srcNames;
     }
+
+    /**
+     * Get relative path when a bal file is inside a directory.
+     *
+     * @param sourcePath     source path
+     * @param patternMatcher pattern matcher string
+     * @param sourceName     source name
+     * @param fileName       file name
+     * @return relative path
+     */
+    public static String getDirectoryRelativePath(String sourcePath, String patternMatcher, String sourceName,
```

Comment:
What do you mean by `patternMatcher` here? AFAIR this the reference location of the source that we are rectifying, right? we can think of using an appropriate arg name instead.

Reply:
Yes, patternMatcher refers to "reference location" in one instance and "reference type" in another instance. I think renaming patternMatcher to "**reference**" would work fine. WDYT?

Reply:
+1 for `reference`/`referencePath`

Reply:
Renamed to `reference`.